nexus/src/app/rack.rs

```diff
@@ -208,6 +234,271 @@ impl super::Nexus {
             self.background_tasks.activate(task);
         }
 
+        // TODO - levon
```

Comment:
@bnaecker and @zephraph, the logic added for applying the initial rack network configuration has been added here. It's very much a "make it work with minimal blast radius" solution and I would like to put this in a saga in a follow up issue, mostly concerned with unblocking upstream work now.